source/command-logging-and-monitoring/tests/README.rst

```diff
@@ -0,0 +1,57 @@
+.. role:: javascript(code)
+  :language: javascript
+
+==============================
+Command Logging and Monitoring
+==============================
+
+.. contents::
+
+--------
+
+Testing
+=======
+
+Automated Tests
+^^^^^^^^^^^^^^^
+There are tests in the `Unified Test Format <../../unified-test-format/unified-test-format.rst>`__ for both logging and
+monitoring in `/logging </logging>`_ and `/monitoring </monitoring>`_, respectively.
+
+
+Prose Tests
+~~~~~~~~~~~
+Drivers MUST implement the following logging prose tests. These tests require the ability to capture log message data in a
+structured form as described in the 
+`Unified Test Format specification <../../unified-test-format/unified-test-format.rst#expectedLogMessage>`__.
+
+Note: the following tests mention string "length"; this refers to length in terms of whatever unit the driver has chosen
+to support for specifying max document length as discussed in the 
+`logging specification <../../logging/logging.rst#truncation-of-large-documents>`__.
+
+*Test 1: Default truncation limit*
+
+1. Configure logging with a minimum severity level of "debug" for the "command" component. Do not explicitly configure the max document length.
+2. Construct an array ``docs`` containing the document ``{"x" : "y"}`` repeated 100 times.
+3. Insert ``docs`` to a collection via ``insertMany``.
+4. Inspect the resulting "command started" log message and assert that the "command" value is a string of length 1000 + (length of trailing ellipsis).
+5. Inspect the resulting "command succeeded" log message and assert that the "reply" value is a string of length <= 1000.
+6. Run ``find()`` on the collection where the document was inserted.
+7. Inspect the resulting "command succeeded" log message and assert that the reply is a string of length 1000 + (length of trailing ellipsis).
```

Comment:
minor: Is there a value in having the second test, as opposed to a parameterizing the first one?